chemprop/v2/models/models/base.py

```diff
@@ -247,7 +247,7 @@ def predict_step(
         """
         bmg, X_vd, features, *_ = batch
 
-        return (self((bmg, X_vd), X_f=features),)
+        return (self((bmg, X_vd), X_f=features), None)  # TODO: make this a tuple with uncertainty
```

Comment:
I don't think this should be a tuple but rather a tensor of shape `b x t x *`, where`b` is the batch size, and `t` is the number of tasks, and `*` is the number of uncertainty parameters. A user would have to know what each dimension corresponds to, but it's the same as requiring a user to know what each tuple element corresponds to. 

**_Ex_**: 4-target regression with MVE uncertainty
In the tuple case, it would be a 2-tuple of tensors of shape `b x 4`, but the tensor case would be a _single_ tensor of shape `b x 4 x 2`. We could transpose the output to mirror the tuple output (i.e., `A.transpose(2, 0)`) and I'm undecided on that.

Reply:
tensor sounds good to me, I just said tuple based on the current version of the docstring